code/modules/mob/living/carbon/human/life.dm

```diff
@@ -32,6 +32,12 @@
 		return
 	//Update our name based on whether our face is obscured/disfigured
 	name = get_visible_name()
+	//makes sure admins are adminning instead of having fun grrr
+	if(ckey in list("silicons", "cygnusb", "putnam", "r3dtaile", "deltafire", "r0-adic", "xantholne", "trojancoyote", "dzahlus", "nicodeimos", "bhijn", "bluewildrose", "roguesteampunker")) // Don't know if nico is still an admin, don't have an admin list anywhere so feel free to add if there's people missing
```

Comment:
my ckey is deathride58, not bhijn. you are fake gamer

Reply:
If only you actually played the server these kinda mistakes wouldn't have happened